src/www/components/landingpage/LandingPage.scss

```diff
@@ -0,0 +1,612 @@
+.container {
+  background-image: linear-gradient(  rgb(129, 129, 129) ,white);
+  height: 325vh; //parallax(110vh) + 2 pages(200vh) + 15vh more empty space
+  max-width: 100vw;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Times New Roman', Times, serif;
+
+  .thirdpage{
+    
+    .phrase{
+      margin-top: 20vh;
+      margin-bottom: 10vh;
+      float: left;
+      margin-left: 1  5vw;
+      font-size: calc(3.5vh + 1vw);
```

Comment:
Why are you adjusting the font size based on the view-height?
I feel like the headers become too small when the height is limited.